sdk/samples/iot/hub/src/paho_iot_hub_twin_example.c

```diff
@@ -231,20 +211,19 @@ static az_result read_configuration_and_init_client()
 
 // Read OS environment variables using stdlib function
 static az_result read_configuration_entry(
-    const char* name,
     const char* env_name,
     char* default_value,
     bool hide_value,
     az_span buffer,
     az_span* out_value)
 {
-  printf("%s = ", name);
-  char* env = getenv(env_name);
+  printf("%s = ", env_name);
+  char* env_var = getenv(env_name);
```

Comment:
nit: I would call this `env_value`. `var` still seems like it could be the name or value.